util/find-doc-nits

```diff
@@ -229,6 +375,14 @@ sub check {
         unless $contents =~ /=for comment generic/
             or $filename =~ m@man[157]/@;
 
+    option_check($id, $filename, $contents)
```

Comment:
You probably should do a single if statement for all three ```name_synopsis``` ```option-check``` and ```functionname_check``` calls.

Reply:
Okie